pkg/dashboards/dashboard_writer.go

```diff
@@ -11,8 +11,21 @@ type DashboardWriter struct {
 }
 
 type DashboardResult struct {
-	builder dashboard.Builder
-	err     error
+	builder   dashboard.Builder
+	component string
+	err       error
+}
+
+func NewDashboardResult(builder dashboard.Builder, err error) DashboardResult {
+	return DashboardResult{
+		builder: builder,
+		err:     err,
+	}
+}
+
```

Comment:
Could you please add a component on for this method? 
For example, It will be useful for new contributors to know that the component name will be the folder name where the dashboards will be stored :)